docs/conformance.md

```diff
@@ -65,7 +81,19 @@ make conformance
 To run without outside connectivity tests add the argument:
 
 ```bash
-make conformance SKIP_OUTSIDE_CONN_TESTS=true
+SKIP_OUTSIDE_CONN_TESTS=true make conformance
+```
+
+To focus on specific tests pass KUBEVIRT_E2E_FOCUS argument:
+
+```bash
+KUBEVIRT_E2E_FOCUS=sig-network make conformance
```

Comment:
it should be `"\\[sig-network\\]"` 
not sure if one `\` is needed or two if you can check please?
(its little tricky, from what i saw in the past)

it will be good to check it with that `[` to find the right syntax as well, because in the `make functest` for example, its sensitive as well to the number of `\` used

Reply:
I tested it directly with `ginkgo` binary and with `make functest` `sig-network`, `\[sig-network\]` and `\\[sig-network\\]` works well and give the same results

Reply:
Please test it end to end (via `make conformance`),
i saw that when testing stuff like this end to end, have sometimes different results than just testing directly
(the scripts in the middle did some effect)

Reply:
Due to how `make conformance` works (runs on bazel-server container using hack/dockerize script) 
it seems that it handles escape characters differently then using `make functest`.
Probably because it passes few more shells with double quotes comparing to `make functest`.

I am not sure its possible to make it work like on `make functest` without bigger change
that I prefer not to do as part of this PR.
Passing `sig-network` or `SRIOV` works as expected, I think its good enough for running kubevirt conformance on SRIOV check-up lane.